build/mvc-scenarios.yml

```diff
@@ -0,0 +1,63 @@
+# Proxy scenarios
+
+parameters:
+- name: profile
+  type: string
+  default: ''
+- name: warmup
+  type: number
+  default: 5
+- name: duration
+  type: number
+  default: 15
+- name: connection
+  type: string
+  default: ''
+
+# Scenarios
+- name: scenarios
+  type: object
+  default:
+  - displayName: Json Mvc
+    arguments: --scenario MvcJson $(jsonJobs) --property displayname="Json Mvc (Hello world)"
```

Comment:
What's the value of jsonJobs? In master it still points to an old format json file

Reply:
It should be a new variable with https://github.com/aspnet/Benchmarks/blob/bfa082b093d46f3f9c336d188a94d56965345250/src/BenchmarksApps/Mvc/benchmarks.mvcjson.yml#L53